crates/storage-query-datafusion/src/partition/row.rs

```diff
@@ -8,35 +8,48 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use crate::table_util::format_using;
+use restate_types::partitions::state::MembershipState;
+use restate_types::{Version, partition_table::Partition};
 
 use super::schema::PartitionBuilder;
-use itertools::{Itertools, Position};
-use restate_types::{Version, cluster::cluster_state::RunMode, partition_table::Partition};
+use crate::table_util::format_using;
 
-#[inline]
 pub(crate) fn append_partition_rows(
```

Comment:
Maybe rename this to `append_partition_row` since now it only append one row per partition unlike before.